src/modules/capabilities.json

```diff
@@ -0,0 +1,71 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+      "title": "Capabilities",
+      "version": "0.0.0",
+      "description": "The Capabilities module provides information about which discreet unit of functionality is enabled for the apps."
+    },
+    "methods": [
+        {
+            "name": "request",
+            "summary": "Requests grants for all capability/role combinations in the roles array.",
+            "params": [
+                {
+                    "name": "grants",
```

Comment:
Does this really need to be specifically grants? What happens when an app requests a capability that doesn't require a grant?

Reply:
Yeah, I've been debating this exact question. I ended up calling it grants, because calling `request` will only ever change if a grant occurs. You can't request that a capability become supported, available, or permitted.

Apps should be written to request _all_ capabilities they need, but after calling this method, the only actual changes are the results of user grants.